posthog/models/feature_flag.py

```diff
@@ -109,6 +110,25 @@ def conditions(self):
         "Each feature flag can have multiple conditions to match, they are OR-ed together."
         return self.get_filters().get("groups", []) or []
 
+    @property
+    def _payloads_by_boolean(self):
+        return self.get_filters().get("payloads", {}) or {}
+
+    def get_payload_by_boolean(self, val: bool) -> Optional[object]:
+        if val:
+            return self._payloads_by_boolean.get("true", None)
+        else:
+            return self._payloads_by_boolean.get("false", None)
```

Comment:
Had this question in the spec: The 'false' payload doesn't really make sense to me here? Effectively means the flag is inactive or disabled right?

Reply:
Yeah, I was thinking it's still useful to have a configurable payload when the flag is evaluating to false

Reply:
Removed